lightkurve/tests/test_targetpixelfile.py

```diff
@@ -734,3 +735,20 @@ def test_estimate_background():
     bg = tpf.estimate_background(aperture_mask='all')
     assert_array_equal(bg.flux.value, 100)
     assert bg.flux.unit == tpf.flux.unit / u.pixel
+
+
+def test_fluxmode():
+    """This should verify the median flux use in an aperture"""
+    tpf = read(filename_fluxmode)
+    aper = np.zeros(tpf.shape[1:], dtype=bool)
+    aper[4:8, 4:7] = True
+    lc_n = tpf.extract_aperture_photometry(aperture_mask=aper, centroid_method='moments')
+    lc_sum = tpf.extract_aperture_photometry(aperture_mask=aper, flux_median="sum", centroid_method='moments')
+    lc_med = tpf.extract_aperture_photometry(aperture_mask=aper, flux_median="median", centroid_method='moments')
+    lc_mean = tpf.extract_aperture_photometry(aperture_mask=aper, flux_median="mean", centroid_method='moments')
```

Comment:
You'll want to replace `flux_median` with `flux_method` in the lines above.  The tests are likely failing because of this.

Reply:
This is outdated -- ignore.

Reply:
Done